### PR TITLE
Updated Readme

### PR DESCRIPTION
**Pull request by MatthewHightech**
Added info on downloading env file and deploying to production.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -82,12 +83,18 @@ Nebo is a being of many talents. Abilities/roles include:
     DEV_MODE=<production | development>
     ```
     * If `DEV_MODE` is set to `development` you will be able to test various commands without requiring _all_ env vars to be set to non-blank values
+<<<<<<< HEAD
```
Comment (evanjohnso) [inline README.md:86]: think there's some leftovers here and `line 90`

Comment (MatthewHightech) [reply README.md:86]: Ahhh thanks haha. I always miss those...

Comment (evanjohnso) [reply README.md]: no matter what IDE i'm building in i always go back to VS Code for merge conflicts. idk why, i just like their little helper UI thingy-mo-bobber

**Review by evanjohnso — APPROVED** [review]

File: README.md [hunk 2]
```diff
@@ -66,7 +66,8 @@ Nebo is a being of many talents. Abilities/roles include:
 6. It should then prompt `Found machine/(project_name) Link to it? (Y/n)` Choose Y, and you should be running now on http://localhost:3000  
 
 ### Run locally
-1. Create a `.env` file for local development
+1. Download `nebo.env` from SSEng in 1password
```
Comment (chrispellett) [inline README.md:69]: Please add href to the item directly

File: README.md [hunk 3]
```diff
@@ -66,7 +66,8 @@ Nebo is a being of many talents. Abilities/roles include:
 6. It should then prompt `Found machine/(project_name) Link to it? (Y/n)` Choose Y, and you should be running now on http://localhost:3000  
 
 ### Run locally
-1. Create a `.env` file for local development
+1. Download `nebo.env` from SSEng in 1password 
+2. Create a .env file for local development
```
Comment (chrispellett) [inline README.md:70]: Is this needed given the step you just added?

File: README.md [hunk 4]
```diff
@@ -115,10 +116,10 @@ go test ./...
...
     ```
     * Vercel requires secrets are set using all lower kabob case so `THIS_EXAMPLE` becomes `this-example`
-2. Deploy
-    ```sh
-    vercel --prod
-    ```
+
+3. Make sure you've checked out Master and have pulled all the changes you want to deploy.
+4. Run `vercel` to deploy to staging and test the application before deploying it to production.
+5. Deploy using `vercel --prod`
```
Comment (chrispellett) [inline README.md:122]: Should there be a step to "revert" any localhost URLs in slack?

**Review by chrispellett — CHANGES_REQUESTED** [review]

**Review by chrispellett — APPROVED** [review]

**Pull request merged**